web/src/components/welcome-modal/welcome-modal.tsx

```diff
@@ -2,8 +2,9 @@ import * as React from 'react';
 import { useEffect, useState } from 'react';
 import { Redirect } from 'react-router';
 import { Link } from 'react-router-dom';
-import { pick } from 'lodash';
```

Comment:
Importing lodash functions like this was causing us to have a much bigger dependancy than we needed.